wal/reader_test.go

```diff
@@ -41,7 +41,7 @@ type reader interface {
 	Offset() int64
 }
 
-type record struct {
+type rec struct {
```

Comment:
Why the change?

Reply:
this package is importing `"github.com/prometheus/tsdb/record"`, so we get `record redeclared...` without renaming the struct type here